Makefile

```diff
@@ -0,0 +1,5 @@
+help:
+	@echo "clean-build - compile sharding manager contract"
+
+compile-smc:
```

Comment:
I'm thinking that it can be changed to something like this
```
compile_script = tools/vyper_compile_script.py
contract = sharding/contracts/validator_manager.v.py
contract_json = sharding/contracts/validator_manager.json

compile-smc: $(contract_json)

$(contract_json): $(contract)
    python $(compile_script) $(contract)
```
to avoid redundant compilation with `target:prequiste`.